scrapers/scrape_vs.py

```diff
@@ -37,5 +66,4 @@
         dd.recovered = sum(r['Nb de nouvelles sorties'] for r in rows[:i+1])
     if not is_first:
         print('-' * 10)
-    is_first = False
```

Comment:
I think this is still needed, should the condition above (line 35) be false, then `is_first` must be set here.

Reply:
Whoops, you're right! Reverted this change in d9dcf5c